server/apidata/apidata.go

```diff
@@ -0,0 +1,238 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package apidata
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"decred.org/dcrdex/dex"
+	"decred.org/dcrdex/dex/encode"
+	"decred.org/dcrdex/dex/msgjson"
+	"decred.org/dcrdex/server/comms"
+	"decred.org/dcrdex/server/db"
+	"decred.org/dcrdex/server/matcher"
+)
+
+const (
+	// DefaultCandleRequest is the number of candles to return if the request
+	// does not specify otherwise.
+	DefaultCandleRequest = 50
+	// CacheSize is the default cache size. Also represents the maximum number
+	// of candles that can be requested at once.
+	CacheSize = 1000
```

Comment:
Can unexport?

Reply:
I thought it might come in handy for apps like dcrdata to keep a cache from the websocket feed on the client side.